harness/determined/cli/user.py

```diff
@@ -78,7 +78,7 @@ def change_password(parsed_args: Namespace) -> None:
     elif parsed_args.user:
         username = parsed_args.user
     else:
-        username = authentication.must_cli_auth().get_session_user()
+        username = client.get_session_username()
```

Comment:
```must_cli_auth``` raising an exception for a CLI using auth seems rather unexpected

Reply:
python sdk has one singleton, cli has another. ideally we'd move to the one for python sdk completely...